addons/cookoff/functions/fnc_handleDamage.sqf

```diff
@@ -16,21 +16,15 @@
 #include "script_component.hpp"
 
 params ["_simulationType", "_thisHandleDamage"];
-_thisHandleDamage params ["_vehicle", "", "_damage", "_source", "_ammo", "_hitIndex", "_shooter"];
+_thisHandleDamage params ["_vehicle", "", "_damage", "_source", "_ammo", "_hitIndex", "_shooter", "_hitpoint", "_oldDamage"];
```

Comment:
Don't like `_oldDamage` here. It's not a parameter of the event and should not be abused to `private` a variable. There could come a point where the parameter is used by the event!